### PR TITLE
Fix U-Boot script capability & add an example

### PR DESCRIPTION
**Pull request by otyshchenko1**

### CODE REVIEW

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/files/0001-Enable-everything-needed-source-command-to-work.patch [hunk 1]
```diff
@@ -0,0 +1,52 @@
+From e94f84b9febb4ff4e41f758a58434fb0641b7642 Mon Sep 17 00:00:00 2001
+From: Oleksandr Tyshchenko <oleksandr_tyshchenko@epam.com>
+Date: Fri, 2 Feb 2018 21:42:51 +0200
+Subject: [PATCH] Enable everything needed source command to work
+
+Fix for the cmd_source was taken here:
+https://lists.denx.de/pipermail/u-boot/2015-July/221152.html
```
Comment (aanisov) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/files/0001-Enable-everything-needed-source-command-to-work.patch]: Why didn't you just backport mentioned patch?

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/files/0001-Enable-everything-needed-source-command-to-work.patch [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+--- a/include/configs/rcar-gen3-common.h
++++ b/include/configs/rcar-gen3-common.h
+@@ -125,4 +125,8 @@
+ #define CONFIG_SYS_MALLOC_LEN		(1 * 1024 * 1024)
+ #define CONFIG_SYS_BOOTMAPSZ		(8 * 1024 * 1024)
+ 
++#define CONFIG_CMD_SOURCE
++#define CONFIG_CMD_ECHO
++#define CONFIG_IMAGE_FORMAT_LEGACY /* enable also legacy image format */
++
```
Comment (aanisov) [inline recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/files/0001-Enable-everything-needed-source-command-to-work.patch]: This is rather a build configuration change. Please separate it from fix.

Comment (otyshchenko1) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/files/0001-Enable-everything-needed-source-command-to-work.patch]: done

File: recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/files/0001-Enable-everything-needed-source-command-to-work.patch (hunk 1 above)
Comment (otyshchenko1) [reply recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-bsp/u-boot/files/0001-Enable-everything-needed-source-command-to-work.patch]: done

**Comment by otyshchenko1** [comment]
@iartemenko pls merge

**Pull request merged**